### DISCUSSION TITLE
What if I am unable to finalize the trade on time or at all (due to bank AML for example)

### DISCUSSION THREAD

**Discussion by slrslr** [bisq-network/bisq#6991] [Q&A]
What happen if i can not finalize Bisq trade due to my bank holds the incoming payment and wanting to prove source of the money?

I believe that i should not loose any money (beside transaction fees) due to such situation, but what is the reality? At https://bisq.wiki/Table_of_penalties seems to be mentioned tens of % penalties, but that can be a lot of money in case of large amount of money which does not seem fair since the delay is not caused by me but by the bank AML policies.

I wanted to also ask which proof of funds source i can supply so it is accepted and it reveals minimum private data?

**Comment by pazza83** [comment]
Have a look at: https://bisq.wiki/Trading_rules

> Trade rules also help with the following:
> 
> Promoting good behavior from traders
> Penalizing bad behavior from traders
> Promoting good payment accounts of traders
> Penalizing bad payment accounts of traders
> Put the 'wrong doer' in a worse position and the person 'not at fault' in a better position

A user can be penalized if they are using a bad payment account. 

> What happen if i can not finalize Bisq trade due to my bank holds the incoming payment and wanting to prove source of the money?

The best way to mitigate any penalty is to communicate with your peer early to let them know the situation in trader chat and open mediation. Most penalties in these can be avoided to effective communication. Most peers will understand that banks can be a pain. Maybe an alterative method of payment or bank account can be agreed in mediation to complete the trade.

If your bank is 'holding' the buyer's funds from the transaction you should ask them to either credit your account or refund the sender as soon as possible.

**Reply by slrslr** [reply]
Thank you. Which proof of funds source is safe to supply my bank so they accept the payment and it reveals minimum private data?

**Reply by pazza83** [reply]
No problems, I suppose the proof of funds issue is a question for your bank.
